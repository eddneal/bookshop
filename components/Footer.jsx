import React from 'react';

export default () => (
  <footer>
    <p>Copyright Â© Bookshop 2018</p>
    <style jsx>{`
    footer {
      padding: 30px;
      text-align: right;
      background-color: #000;
      color: #fff;
    }
  `}</style>
  </footer>
);
