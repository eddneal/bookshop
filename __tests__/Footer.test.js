import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../components/Footer';

describe('With Enzyme', () => {
  it('Footer shows copyright info', () => {
    const app = shallow(<Footer />);

    expect(app.find('p').text()).toEqual('Copyright Â© Bookshop 2018');
  });
});

describe('With Snapshot Testing', () => {
  it('Matches Snapshot', () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});