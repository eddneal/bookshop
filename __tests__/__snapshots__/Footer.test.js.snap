// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Matches Snapshot 1`] = `
<footer>
  <p>
    Copyright Â© Bookshop 2018
  </p>
</footer>
`;
